api/config/dex/config_request_test.go

```diff
@@ -369,6 +370,23 @@ func TestValidate(t *testing.T) {
 		})
 	})
 
+	t.Run("allowed_groups (saml)", func(t *testing.T) {
+		valid := map[string][]string{
+			"one group":  {"admins"},
+			"two groups": {"admins", "ops"},
+			"no groups":  {}, // should this warn?
```

Comment:
I think throwing an error or warning if they try to set empty groups is a good idea 👍

Reply:
Warnings aren't there, but we can make it error. Thinking about it, it seems worthwhile. :+1:

Reply:
OK, turns out `allowed_groups = []` is treated the same way as *unset*. So, the user cannot set this. I thus don't think we'll need an error. It cannot happen. I'll merge this 😃